lib/drupal.js

```diff
@@ -32,7 +32,7 @@ module.exports = function(grunt) {
   };
 
   module.majorVersion = function() {
-    var version = grunt.config('drupal.drupal-version') || '',
+    var version = process.env.GDT_DRUPAL_CORE || grunt.config('drupal.drupal-version') || '',
```

Comment:
Our generators don't currently set `drupal-version`, so among other places, GDT's tests use this variable.